dbms/src/Common/TiFlashMetrics.h

```diff
@@ -671,7 +671,8 @@ namespace DB
     M(tiflash_system_seconds,                                                                                                       \
       "system calls duration in seconds",                                                                                           \
       Histogram,                                                                                                                    \
-      F(type_fsync, {{"type", "fsync"}}, ExpBuckets{0.0001, 2, 20}))
+      F(type_fsync, {{"type", "fsync"}}, ExpBuckets{0.0001, 2, 20}))                                                                \
+    M(tiflash_rn_delta_index_cache, "", Counter, F(type_hit, {"type", "hit"}), F(type_miss, {"type", "miss"}))
```

Comment:
```suggestion
    M(tiflash_storage_delta_index_cache, "", Counter, F(type_hit, {"type", "hit"}), F(type_miss, {"type", "miss"}))
```

I think `tiflash_storage_` is a better prefix